engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -787,6 +787,24 @@ object Main {
     runMain(mainModule, Some(file), additionalArgs)
   }
 
+  /** If the stack trace ends with `Assertion_Error`, drops the internal assertion frames.
+    */
+  private def dropAssertionError(
```

Comment:
A not-so-nice workaround for a pretty assertion error message. An alternative is to replace `Panic.throw` in `Runtime.assert` with:
```
            dropped_stack = get_stack_trace.drop (First 2)
            IO.print_err <| "Assertion Error: " + message
            dropped_stack.each stack_frame->
                IO.print_err <| "        " + stack_frame.to_display_text
            System.exit 1
```

But I assume we don't want to do a hard-exit, but rather throw a standard Panic.

Suggestions?

Reply:
Dropped in 3fc7cc899864656937a77955fd06b82ce76cf6a2